lib/presentation/screens/details_screen/bloc/details_screen_state.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:equatable/equatable.dart';
+import 'package:marvelapp_flutter/data/model/response_models/series.dart';
+import '../../../../data/model/response_models/character.dart';
+
+enum DetailsScreenStatus { initial, loading, success, error }
```

Comment:
With this you need to have an empty `Character ` with null fields which does not make sense. I reccomend using nullable values like this:

```

class DetailsScreenState extends Equatable {
  final Bool loading; // if true - this is loading state
  final Character? character; // if null - it is an error state with "No details present" message
  final String? error; // in non-null - it is an error state with 'error' message
  final List<Series>? series; // do not show series section if it is null
```

Reply:
done